pkgs/top-level/all-packages.nix

```diff
@@ -21963,11 +21962,6 @@ with pkgs;
     callPackage ../applications/networking/cluster/terraform-providers {}
   );
 
-  terraform-provider-libvirt = callPackage ../applications/networking/cluster/terraform-provider-libvirt {};
-
-  terraform-provider-ibm = callPackage ../applications/networking/cluster/terraform-provider-ibm {};
-
-
```

Comment:
Where did those go?

Reply:
Hmm, overlooked when asking @ryantm 

This is verbatim from the cherry-picked commit.